bundles/org.openhab.binding.powermax/src/main/java/org/openhab/binding/powermax/internal/connector/PowermaxConnector.java

```diff
@@ -63,11 +62,17 @@ protected void cleanup() {
         }
 
         if (output != null) {
-            IOUtils.closeQuietly(output);
+            try {
```

Comment:
I think using `closeQuietly` is wrong here. The description clearly says, that it should only be used when normal processing failed (e.g. to close after an exception has been thrown and we are in a `finally` block), This is clearly not the case here. I would prefer to log something in case closing failes.

Reply:
All bindings are doing that like that !
I have already 12 PRs to update in this case.
Is DEBUG level ok for you ?